app/assets/stylesheets/common/base/user.scss

```diff
@@ -171,7 +171,7 @@
       }
 
       .groups {
-        margin-left: 10px;
+        margin-left: 0px;
```

Comment:
Thanks, this is a good change. We can probably remove this line altogether, by default the margin-left here would be 0.

Reply:
Alright, will remove the line.

Reply:
It doesn't look like the commit to remove the line made it through.

Reply:
Let me check quickly

Reply:
Committed, try again.